shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -179,9 +185,11 @@ private void cacheBlock(long blockId, WorkerNetAddress dataSource, URIStatus sta
     try (CloseableResource<BlockWorkerClient> blockWorker =
         mFsContext.acquireBlockWorkerClient(dataSource)) {
       blockWorker.get().cache(request);
+      return true;
     } catch (Exception e) {
       System.out.printf("Failed to complete cache request for block %d of file %s: %s", blockId,
           status.getPath(), e);
+      return false;
```

Comment:
I prefer using exceptions to propagate the result of the operation if it is a failure. We can do:

```suggestion
    try (CloseableResource<BlockWorkerClient> blockWorker =
        mFsContext.acquireBlockWorkerClient(dataSource)) {
      blockWorker.get().cache(request);
    } catch (Exception e) {
      throw new RuntimeException(String.format("Failed to complete cache request for block %d of file %s: %s", blockId,
          status.getPath(), e), e);
     }
```

and catch the exception inside the `load` method, which should be the sole method responsible for converting exceptions into error messages and printing them out.